### ISSUE TITLE
Progress bar or other progress indicators for possibly lengthy tasks like `aaa`

### ISSUE THREAD

**Issue by Rot127** [rizinorg/rizin#4285]
**Is your feature request related to a problem? Please describe.**

While waiting for a 100MB+ file to be analyzed I wished for a progress bar to help maintain a peaceful mind.

It doesn't need to provide a estimated time when it's done (which is impossible for most tasks anyways). But watching some increasing numbers would be nice.

**Describe the solution you'd like**

E.g.

- When all flags with `sym.` are analyzed, we could have a progress bar showing the total numbers of symbols already processed.
- For emulation based analysis, a list of executed known and newly discovered functions would be nice.

Basically adding an API to show a linear progress for which 100% is known (progress bar).
Or update some numbers of the current task running.

**Describe alternatives you've considered**

Not adding it, because it consumes too many resources.

**Additional context**

Though with the introduction of RzIL long running task might become a common thing.
Providing some feedback, even if it doesn't include an estimated time when done.
